web-console/future-openshift/chargeback/chargeback.md

```diff
@@ -2,20 +2,27 @@
 
 ## Reports
 ![](img/reports-list.png)
-- The chargeback page contains charts showing several cost metrics as well as a list of all chargeback reports
 
 ## Creating a Report
 ![](img/create-report.png)
-- Similar to other pages, a dismissible sidebar enables users to select from several sample reports that will be filled into the editor
+- The `Report Query` selector contains a list of the queries that could be used to create a report.
+- The namespace could be selected, or potentially filled in automatically depending on the selected query.
+- Checking `Run Immediately` could make the start and end dates required fields if applicable.
 
-![](img/pending-report.png)
-- One-off reports should not have a value filled in for the schedule column
+![](img/create-report-2.png)
+- In addition to creating an unscheduled report, users would be able to choose from a number of available periods or manually enter a CRON expression.
 
-## Report Details
-![](img/report-details.png)
-- Each report page contains information about the report and its scheduling as well as a table that contains the actual report itself.
+![](img/create-report-3.png)
```

Comment:
Hey Chris, is this indented fields for the month,hour,min,sec a pattern that's used in the console in other places? We had something similar in the Create VM but pulled it b/c it didn't seem needed and we didn't see it anywhere else.

Reply:
It would appear in the cron-jobs flow as well. @spadgett seemed interested in making it a pattern in conjunction with the `schedule` dropdown.